napari/_vispy/canvas.py

```diff
@@ -634,6 +639,33 @@ def _add_overlay_to_visual(self, overlay: Overlay) -> None:
             vispy_overlay.node.parent = self.view.scene
         self._overlay_to_visual[overlay] = vispy_overlay
 
+    def _calculate_view_direction(self, event_pos) -> List[float]:
+        """calculate view direction by ray shot from the camera"""
+        x, y = event_pos
+        w, h = self.size
+        nd = self.viewer.dims.ndisplay
+
+        transform = self.view.camera._scene_transform
+        p0 = transform.imap([x, y, 0, 1])  # map click pos to scene coordinates
+        p1 = [
+            w / 2,
+            h / 2,
+            -1e10,
```

Comment:
Any reason why we shouldn't use actual infinite? Or anything bigger than `1e10`?